scripts/continuous_integration/common/drake_bazel_external

```diff
@@ -35,6 +35,7 @@ bazel version
 
 pushd drake_bazel_external
 
-bazel test ...
+# TODO uncomment once drake_bazel_external has a supported implementation
+# bazel test ...
 
 popd
```

Comment:
I would be inclined to delete the whole file and rely on the git history to keep a copy.